tests/testthat/test-lightgbm.R

```diff
@@ -700,3 +700,74 @@ test_that("multi_predict() predicts classes if 'type' not given ", {
     expect_s3_class(pred_tbl[[".pred_class"]], "factor")
     expect_true(all(as.character(pred_tbl[[".pred_class"]]) %in% levels(penguins[["sex"]])))
 })
+
+test_that("lightgbm with case_weights",{
+  skip_if_not_installed("lightgbm")
+  skip_if_not_installed("parsnip")
+
+  suppressPackageStartupMessages({
+    library(lightgbm)
+    library(parsnip)
+  })
+
+  set.seed(1234L)
+
+  # Following example here:
+  # https://github.com/microsoft/LightGBM/blob/638014d5c56fb92396bd55f344a47e3f651a3cd4/R-package/demo/weight_param.R
+
+  # Setup small weights
+  weights1 <- rep(1e-5, 6513L)
+  weights2 <- rep(1e-5, 1611L)
+
+  data(agaricus.train, package = "lightgbm")
+  dtrain <- lgb.Dataset(agaricus.train$data, label = agaricus.train$label, weight = weights1)
+  train <- data.frame(as.matrix(agaricus.train$data))
+  train$label<-agaricus.train$label
```

Comment:
Could you do a pass through these tests enforcing [tidy style](https://style.tidyverse.org/syntax.html)?
```suggestion
  train$label <- agaricus.train$label
```

Namely, spaces before and after infix operators.

Reply:
I hope I got them all this time.